README.md

```diff
@@ -1,9 +1,54 @@
-# Jarivs Data Engineering Training
+# Kelvin Duong . Jarvis Consulting
 
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+I graduated with a bachelor's degree specializing in Computer Science from the University of Toronto Scarborough. The coursework has exposed me to an extensive set of programming and interpersonal skills. It has also allowed me to develop a strong understanding of data structures and algorithms through practical assessments. I joined Jarvis as a Data Engineer to improve on my software development abilities and to learn the newest technologies used in the field. I applied my deep knowledge of Computer Science fundamentals to the projects while collaborating with a team in an Agile/Scrum environment. I am passionate about learning new technologies and design patterns that will allow me to grow as a Data Engineer. During my free time, I play competitive videogames and practice on the piano. It helps me to build on my communication skills and to work well in a team to accomplish a common goal.
```

Comment:
I graduated with a bachelor's degree specializing in Computer Science from the University of Toronto Scarborough. The coursework has exposed me to an extensive set of programming and interpersonal skills. It has also allowed me to develop a strong understanding of data structures and algorithms through practical assessments. I joined Jarvis as a Data Engineer to improve my software development abilities and learn the newest technologies used in the field. I applied my deep knowledge of Computer Science fundamentals to the projects while collaborating with a team in an Agile/Scrum environment. I am passionate about learning new technologies and design patterns that will allow me to grow as a Data Engineer. During my free time, I play competitive videogames and practice on the piano. It helps me build on my communication skills and work well in a team to accomplish a common goal.

fixed minor grammar issues for you